CONTRIBUTING.rst

```diff
@@ -58,7 +58,7 @@ of the core developers before it is merged into the main repository.
 #. Check for open issues or open a fresh issue to start a discussion around a
    feature idea or a bug.
 #. If you feel uncomfortable or uncertain about an issue or your changes, feel
-   free to email sphinx-dev@googlegroups.com.
+   free to email the *sphinx-dev* mailing list.
```

Comment:
Why removing this address? How can contributors know it? I think hyperlink is much better.

Reply:
Done